README.md

```diff
@@ -61,13 +61,13 @@ use pyo3::prelude::*;
 use pyo3::wrap_pyfunction;
 
 #[pyfunction]
-/// Formats the sum of two numbers as string
+/// Formats the sum of two numbers as string.
 fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
     Ok((a + b).to_string())
 }
 
-/// This module is a python module implemented in Rust.
 #[pymodule]
+/// A Python module implemented in Rust.
```

Comment:
Actually [the rust style guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md#doc-comments) suggests "Doc comments should come before attributes".